TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -151,16 +181,25 @@ public class OptionsManager
                 ToCheckbox(data, idx: 57, PoliciesTab.DedicatedTurningLanes);
 
                 Options.SavegamePathfinderEdition = LoadByte(data, idx: 58, defaultVal: 0);
+
+                Options.Available = true;
+
                 return true;
             }
             catch (Exception ex) {
                 ex.LogException();
+
+                // even though there was error, the options are now available for querying
```

Comment:
huh .. are you sure about that?

Reply:
Yes. If the load method fails, the options will just have default values but TM:PE will still be fully operable.

The goal was to prevent other mods reading options either prior to options being set, or during save/load, or from main menu.

Reply:
But whats the point of playing with default options?
And what happens if I save the game with those defaults?
And what if another mod makes descisions based on that?

Reply:
If its fine for krzy its fine for me though.

Reply:
I think we shouldn't try to save anything if failed to load, that won't overwrite old data, so in case of a bug in load code it could be loaded after applying hotfix - IIRC game is cloning mod data container so even if you not longer have a mod, data will be migrated to new saves.

Reply:
Prior to this PR, what would the difference be?

If the data fails to load, the `LoadData()` method will still `return false` just like it previously did, which results in `loadingSucceeded = false` at the call site - that currently has no effect on whether `SaveData()` gets invoked later.

I've created an issue to track the need to prevent data save should data fail to load.

Reply:
This PR is ok, you can merge :)

Reply:
https://github.com/CitiesSkylinesMods/TMPE/issues/1412